frameworks/PHP/swoole/swoole-async-pgsql.dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ubuntu:22.04
+
+ENV PHP_VERSION 8.2.10
+ENV SWOOLE_VERSION 5.1.0
+ENV ENABLE_COROUTINE 1
+ENV DATABASE_DRIVER pgsql
+
+RUN apt update > /dev/null \
+    && apt install -y autoconf re2c bison gcc g++ libxml2-dev pkg-config libsqlite3-dev libssl-dev libpq-dev zlib1g-dev make curl > /dev/null \
+    && cd /tmp \
+    && curl -sSL "https://github.com/php/php-src/archive/refs/tags/php-${PHP_VERSION}.tar.gz" | tar xzf - > /dev/null \
+    && curl -sSL "https://github.com/swoole/swoole-src/archive/v${SWOOLE_VERSION}.tar.gz" | tar xzf - > /dev/null \
```

Comment:
Do we need to build PHP in each iteration ?  This will be very slow to build the docker image.
I can understand than you want to compile the Swoole, but not PHP also.